PortFxCop/src/Tools/FileIssues/Program.cs

```diff
@@ -70,35 +82,88 @@ private IEnumerable<PortingInfo> GetRulePortingInfo()
 
         private async Task CreateIssues(IEnumerable<PortingInfo> rulesToPort)
         {
-            IIssuesClient client = GitHub.GetIssuesClient(_options.Token);
-            var existingIssues = await client.GetAllForRepository(_options.RepoOwner, _options.RepoName);
+            var existingIssues = await _issuesClient.GetAllForRepository(
+                _options.RepoOwner,
+                _options.RepoName,
+                new RepositoryIssueRequest
+                {
+                    Filter = IssueFilter.All,   // Get issues whether or not they're assigned to me.
+                    State = ItemState.All       // Get issues whether they're open or closed.
+                });
```

Comment:
Get all issues, not just those I have an interest in (because they're assigned to me, or I created them, or I subscribed to them), and not just the open ones (because if we change the Port Status from "no" to "yes", I'll want to reopen the corresponding closed issue.